src/metadata_tool/sources/github.clj

```diff
@@ -93,17 +94,78 @@
        (if-not (= 404 (:status (ex-data ei#)))
          (throw ei#)))))
 
+(defn content-fn
+  "Returns the contents of file"
+  [org repo path]
+  (try 
+    (:body (http/get
+     (str "https://raw.githubusercontent.com/" org "/" repo "/master/" path)))
+    ; TODO - better exception management here! Only catch 404, throw the others
+    (catch Exception _ "")))
+(def content (memoize content-fn))
+
+(defn folder-fn
+  "Returns the metadata of a folder in GitHub"
+  [org repo path]
+  (try
+    (http/get
+     (str "https://api.github.com/repos/" org "/" repo "/contents/" path))
+    ; TODO - better exception management here! Only catch 404, throw the others
+    (catch Exception _ "")))
+(def folder (memoize folder-fn))
+
+(defn pending-invitations-fn
+  "Returns the list of pending invitations for a given org"
+  [org-name]
+  (try
+    (http/get
+     (str "https://api.github.com/orgs/" org-name "invitations"))
+    ; TODO - better exception management here! Only catch 404, throw the others
+    (catch Exception _ "")))
+(def pending-invitations (memoize pending-invitations-fn))
+
+(defn invite-member-fn
```

Comment:
since this is a `nop` and it's never called, I would not yet include it.

Reply:
It does print out the results, instead of sending actual emails, which is why I'd like to keep it. This way we can run the full test against the master definition of FINOS metadata.

Note that this feature is expected to be rolled out next week.